### PR TITLE
Add a simple README.md file

### PR DESCRIPTION
**Pull request by clare-dsq**
Adding a README.md file that was missing for the project.

### CODE REVIEW

**Review by cleirdu — CHANGES_REQUESTED** [review]
This README is too short. It would be nice to see an example.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Scripts that perform some simple tasks simulating some of
+our very basic day-to-day activities, like sending a greeting
+message to others or checking out the current date and time.
+
+### Greetings
+
+This module sends a simple greeting message with your name
+tagged along at the end, given your name.   
+It wil send out the message "Hi there! -- FirstName LastName"
```
Comment (cleirdu) [inline README.md:11]: This sentence is missing a period at the end

Comment (cleirdu) [inline README.md:11]: This sentence is missing a period at the end

File: README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+##### Example
+
+Calling `greetings("Clare")` will return `"Hi there! -- Clare"`
```
Comment (cleirdu) [inline README.md:15]: This is cool, but please add a couple more examples.

**Review by cleirdu — COMMENTED** [review]
Thanks! Please have a look at the comments

**Comment by clare-dsq** [comment]
All comments resolved, please take another look.

**Review by cleirdu — COMMENTED** [review]
Thanks! There still remains a small problem to be fixed. Good job overall!

**Review by cleirdu — APPROVED** [review]
Nice work! You can now go ahead and merge your changes into the main branch!

**Pull request merged**